src/translations/en/publication.json

```diff
@@ -76,6 +76,8 @@
   },
   "references": {
     "article_number": "Article number",
+    "delete_doi_title": "Change type?",
+    "delete_doi_text": "By removing link to publication you will change from registration to publishing. Do you want to continue?",
```

Comment:
`By removing the link to the publication it will change from being a registration to a new publication` ?

Dette er en rar setning uansett om man skriver den "korrekt". Disse begrepene bør spesifiseres nøyere når vi kommer oss til pilot hvertfall. Kanskje @brinxmat kan komme med noe input?

Reply:
Forslag til en liten språkendring:
`Removing the link to the publication will change it from a registration to a new publication`